src/main/java/com/boardgo/domain/meeting/service/facade/MeetingCommandFacadeImpl.java

```diff
@@ -138,25 +133,35 @@ public void updateMeeting(
         }
     }
 
+    private void validateLimitParticipantCount(MeetingUpdateRequest updateRequest, Long meetingId) {
+        MeetingParticipantSubEntity meetingParticipantSubEntity =
+                meetingParticipantSubQueryUseCase.getByMeetingId(meetingId);
+        if (updateRequest.limitParticipant() <= 1
+                || meetingParticipantSubEntity.getParticipantCount()
+                        > updateRequest.limitParticipant()) {
+            throw new CustomIllegalArgumentException("현재 참여한 인원보다 최대 인원수가 커야합니다.");
+        }
+    }
+
     private String updateImage(
-            MultipartFile imageFile, List<Long> boardGameIdList, MeetingEntity meeting) {
-        // 1. 이미지 파일 수정 X, 보드게임 수정 X
-        // 2. 이미지 파일 수정 X, 보드게임 수정 O, thumbnail 사용자 등록 이미지인 경우
-        if (Objects.isNull(imageFile)
-                && (Objects.isNull(boardGameIdList)
-                        || (Objects.nonNull(meeting.getThumbnail())
-                                && meeting.getThumbnail().startsWith("meeting")))) {
-            return meeting.getThumbnail();
-        } else {
+            MultipartFile imageFile, List<Long> boardGameIdList, MeetingEntity meeting, MeetingUpdateRequest updateRequest) {
+        // 1. 썸네일을 지운 경우 - 기존 썸네일이 사용자가 등록한 이미지
+        // 2. 썸네일이 있는 경우 - 기존 사용자가 등록한 이미지가 있든지 말든지 모두 처리
+        // 3. 기존 썸네일이 사용자가 올린 경우가 아닌 경우 - 보드게임 이미지를 변경해야함
+        if (updateRequest.isDeleteThumbnail() || Objects.nonNull(imageFile) || (meeting.getThumbnail().startsWith("boardgame") && Objects.nonNull(updateRequest.boardGameIdList()))) {
             s3Service.deleteFile(meeting.getThumbnail());
             if (Objects.isNull(boardGameIdList)) {
                 List<Long> meetingBoardGameIdList =
-                        boardGameQueryUseCase.findMeetingDetailByMeetingId(meeting.getId()).stream()
-                                .map(BoardGameByMeetingIdResponse::boardGameId)
-                                .toList();
+                    boardGameQueryUseCase.findMeetingDetailByMeetingId(meeting.getId()).stream()
+                        .map(BoardGameByMeetingIdResponse::boardGameId)
+                        .toList();
                 return registerImage(meetingBoardGameIdList, imageFile);
             }
             return registerImage(boardGameIdList, imageFile);
```

Comment:
이미지 등록하는 `registerImage` 메소드에서 보드게임의 첫번째 Id 만 필요하니까 
``` java
BoardGameEntity boardGameEntity =
                    boardGameQueryUseCase.getById(boardGameIdList.getFirst());
```

필요한 매개변수만 받을 수 있도록 변경해야 될 것같아요! (위의 리뷰에서 말한 매개변수로 보드게임id를 받는것)
```suggestion
                return registerImage(boardGameId, imageFile);
            }
            return registerImage(boardGameId, imageFile);
```

Reply:
검증을 registerImage에서 하고 있어서 first를 또 검증하는 코드를 추가해야해서 일부러 List로 전달했습니다!

Reply:
아 그런데 registerImage 메서드가 List<Long>을 받지 않아도 돼서 수정하겠습니다 ~!